Add missing support to redis sentinel (#141)
openfun/openedx-docker: README.md

Docker compose also allows us to run a complete project in development,
including database services which in production are not run on Docker. See the

> [docker-compose file](./docker-compose.base.yml) for details on each service:

Reviewer: ![](https://media3.giphy.com/media/LRKET0Syb0rDO/giphy.gif)

  proxy other requests to Django.
- **mailcatcher** the email backend

Concerning Redis, it is possible to run a single redis instance (the default choice)
or to run redis with sentinel to simulate a HA instance. Depending the version you want,

> two different docker-compose file have been crafted. 

Reviewer: ```suggestion
two different docker-compose files have been crafted. 
```

> By default you will use the single redis instance, to use redis sentinel you have to set

Reviewer: Avoid default behavior repetition :
```suggestion
To use Redis sentinel you have to set
```

> the environment variable REDIS_TARGET_INSTANCE to `redis-sentinel`:

Reviewer: ```suggestion
the `REDIS_TARGET_INSTANCE` environment variable  to `redis-sentinel`:
```

```bash
$ export REDIS_TARGET_INSTANCE=redis-sentinel
```

> To switch back to the single redis instance, set this same variable to `redis`.

Reviewer: ```suggestion
To switch back to the single redis instance, unset this environment variable:
```
```
$ unset REDIS_TARGET_INSTANCE
```

  proxy other requests to Django.
- **mailcatcher** the email backend

Concerning Redis, it is possible to run a single redis instance (the default choice)
or to run redis with sentinel to simulate a HA instance. 
To use Redis sentinel you have to set the `REDIS_TARGET_INSTANCE` environment variable
to `redis-sentinel`:

```bash

> $ export REDIS_TARGET_INSTANCE=redis-sentinel

Reviewer: ![](https://media0.giphy.com/media/VqPpOjb1rOBBS/giphy.gif)

  proxy other requests to Django.
- **mailcatcher** the email backend

Concerning Redis, it is possible to run a single redis instance (the default choice)

> or to run redis with sentinel to simulate a HA instance. Depending the version you want,

Reviewer 2: ```suggestion
or to run redis with sentinel to simulate a HA instance. Depending on the version you want,
```
